CIP-0146/README.md

```diff
@@ -0,0 +1,220 @@
+---
+CIP: 146
+Title: Multi-signatures wallet registration and discovery

```

Comment:
```suggestion
Title: Multi-signature wallet registration and discovery
```
someone can correct my grammar which is correct here? I have also assumed 'Multi-signature' is a plural

Reply:
@Ryun1 @Scitz0 it's necessary in English & other languages I know to avoid a "double plural".  The prefix `Multi-` makes it a plural so adding the `s` at the end is redundant and therefore incorrect.  It should be an item on someone's to-do list to correct any such mistakes in CIP-1854 😅

Reply:
Updated all references for this spelling error :+1: